src/README.md

```diff
@@ -3,8 +3,7 @@
 This repository includes several documents describing how to perform internal tasks within the [l10n-drivers team](https://wiki.mozilla.org/L10n:Mozilla_Team) at Mozilla.
 
 Topics covered:
-* Manage [Firefox for Android](products/firefox_android/) and [Firefox desktop](products/firefox_desktop/): setting up builds, bootstrap productization settings, and testing.
-* How to [review strings](products/review/review.md) and [perform sign-offs](products/review/signoffs.md) for Firefox desktop and Firefox for Android.
+* How to [review strings](products/review/review.md) and [perform sign-offs](products/review/signoffs.md) for Firefox desktop.
```

Comment:
Technically we're still doing sign-offs for Firefox for Android, until it's completely removed, but I think it's OK to drop the mention already.